app/services/set_decline_by_default.rb

```diff
@@ -0,0 +1,29 @@
+class SetDeclineByDefault
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
+  def call
+    return false unless application_choices.count == 3
```

Comment:
Make `3` a constant somewhere?

Reply:
What happens if the candidate only makes 1 or 2 choices? (My understanding may be wrong here - I'm thinking that 3 is the maximum and that candidates don't have to use all their choices)

Reply:
This logic also needs to check that the choices have all received decisions — there can be fewer than three

Reply:
We updated this, now do not check the number of choices.
As we understood there is no need to check the Max number of choices,
is that correct?
cc @mnacos

Reply:
Yes, we've refactored this to check if there are any application choices awaiting decision, so we don't need any constants and the code works with any number of application choices. The service doesn't need to know anything about the number of application choices allowed elsewhere in the application and shouldn't have any role enforcing these limits.